### DISCUSSION TITLE
RoadMap V5

### DISCUSSION THREAD

**Discussion by zergormagn** [algolia/instantsearch#5108] [General] [answered]
This might be a stupid question, but is there any roadmap available as to what is coming in the next major version?  Any timeline?

**Comment by Haroenv** [comment, marked as answer]
At the moment we have no big features that present the need for a v5 version of InstantSearch.js, which is why there's no plan, however if you search for `@deprecated` and `@major` in the codebase, you'll find some of the items we plan to remove or update when there's a next major version.
